R/clean-wqdata.R

```diff
@@ -102,6 +95,7 @@ clean_wqdata_by <- function(x, max_cv, messages) {
 #' @param remove_blanks Should blanks be removed? Blanks are assumed to be denoted by
 #' a value of `"Blank..."` in the `SAMPLE_CLASS` column. Default `FALSE`
 #' @param messages A flag indicating whether to print messages.
+#' @param FUN The function to use for summaries, e.g. median, mean, or max.
```

Comment:
I like to be specific with the default value:

```suggestion
#' @param FUN The function to use for summaries, e.g. `median`, `mean`, or `max`. Default `mean`
```

Reply:
sounds good. FYI as with the previous PR, the wording was copied from other functions in the package -  you could review ''test-trends.R' for other spots where FUN argument is used and described

Reply:
Thanks - A full review for consistency isn't in scope (at least for me), but we can improve iteratively as we come across things that are easy to fix!

Reply:
OK that makes sense